### PR TITLE
Youtube playlist component, new titles, table of pictures on home page

### PR DESCRIPTION
**Pull request by austinrafter**
Created embedded youtube video playlist component called Youtube. Youtube component is displayed under the SoundCloud component on the home page. Added makefile for quick command runs. NPM install to allow for local development required deleting package-lock.json file and having it recreated to remove outdated version of npm used in inital creation of the project. Added some new pictures to assets that will be displayed in a table on the home page.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
-
+    <h2>
+      Listen to the assholes ramble
+    </h2>
     <sound-cloud></sound-cloud>
+  <h2>
+    You sing and you sing good!
+  </h2>
+    <h3>
+      (on video)
```
Comment (candyapplecorn) [inline src/views/Home.vue:12]: This is just plain text and it's not an actual YouTube video component are you missing commits? That you haven't pushed up yet?

File: src/views/Home.vue [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+    </h3>
+    <youtube></youtube>
```
Comment (candyapplecorn) [inline src/views/Home.vue:14]: 👍

**Comment by candyapplecorn** [comment]
I don't see the makefile in this pull request I also don't see the diff list showing the new YouTube component file

I just looked through the master branch components folder and I don't see the YouTube component in there so I think you forgot to push that up to this pull request branch

I also don't see the new pictures in this diff list

File: src/views/Home.vue (hunk 2 above)
Comment (austinrafter) [reply src/views/Home.vue:14]: were you able to find it or is the component not there? I may have forgotten to add those like a fool haha

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	npm run serve;
+
+deploy:
+	npm run deploy;
+
+docker-build:
+	docker-compose build;
+
+docker-up:
+	docker-compose up;
```
Comment (candyapplecorn) [inline Makefile]: Nice can you also put an example of running to the makefile from a Linux or Mac shell terminal?

like:

```
# to run a makefile script, do the following:
>$: make local-development
```

Cause I can't remember how to run make files and I don't wanna have to look it up when I go to work on this project

File: README.md [hunk 4]
```diff
@@ -46,3 +46,7 @@ i haven't taken the time to write docker commands to publish/deploy this, if tha
 ### austin
 
 *lmk if u want me 2 set u up w/ this so u can make changes 2 the site urself e z p z ;) 30 mins on zoom or less*
+
+
+### changes to cname for github pages on namecheap
+https://www.namecheap.com/support/knowledgebase/article.aspx/9645/2208/how-do-i-link-my-domain-to-github-pages/
```
Comment (candyapplecorn) [inline README.md:52]: 🧠

File: Makefile (hunk 3 above)
Comment (austinrafter) [reply Makefile]: Yeah I'll add that now

**Pull request merged**